theme/styles/global.scss

```diff
@@ -123,3 +123,11 @@
 .hyphens {
   hyphens: auto;
 }
+
+.nav-button-padding {
+  padding: 0.1875rem;
+
+  @include media('md-up') {
+    padding: 0.15rem 0.25rem;
+  }
+}
```

Comment:
I was having to copy and paste this inline for many buttons on the Nav, so thought I would make a util class instead to be able to better keep track of its styling since I think this will probably change during QA.